packages/dev/core/src/Animations/animatable.core.ts

```diff
@@ -0,0 +1,1113 @@
+                        // We need to normalize the weights
+                        normalizer = holder.totalWeight;
+                        const scale = originalAnimation.weight / normalizer;
+                        if (scale !== 1) {
+                            if (originalAnimation.currentValue.scale) {
+                                finalValue = originalAnimation.currentValue.scale(scale);
+                            } else {
+                                finalValue = originalAnimation.currentValue * scale;
+                            }
+                        } else {
+                            finalValue = originalAnimation.currentValue;
+                        }
+
+                        if (originalAnimationIsLoopRelativeFromCurrent) {
+                            if (finalValue.addToRef) {
+                                finalValue.addToRef(originalValue, finalValue);
+                            } else {
+                                finalValue += originalValue;
+                            }
+                        }
+
+                        startIndex = 1;
+                    }
+
+                    // Add up the override animations
+                    for (let animIndex = startIndex; animIndex < holder.animations.length; animIndex++) {
+                        const runtimeAnimation = holder.animations[animIndex];
+                        const scale = runtimeAnimation.weight / normalizer;
+
+                        if (!scale) {
+                            continue;
+                        } else if (runtimeAnimation.currentValue.scaleAndAddToRef) {
+                            runtimeAnimation.currentValue.scaleAndAddToRef(scale, finalValue);
+                        } else {
+                            finalValue += runtimeAnimation.currentValue * scale;
+                        }
+                    }
+
+                    // Add up the additive animations
+                    for (let animIndex = 0; animIndex < holder.additiveAnimations.length; animIndex++) {
+                        const runtimeAnimation = holder.additiveAnimations[animIndex];
+                        const scale: number = runtimeAnimation.weight;
+
+                        if (!scale) {
+                            continue;
+                        } else if (runtimeAnimation.currentValue.scaleAndAddToRef) {
+                            runtimeAnimation.currentValue.scaleAndAddToRef(scale, finalValue);
+                        } else {
+                            finalValue += runtimeAnimation.currentValue * scale;
+                        }
+                    }
+                }
+            }
+            target[path] = finalValue;
+        }
+
+        target._lateAnimationHolders = {};
+    }
+    scene._registeredForLateAnimationBindings.reset();
+}
+
+/** @internal */
+export function RegisterTargetForLateAnimationBinding(scene: Scene, runtimeAnimation: RuntimeAnimation, originalValue: any): void {

```

Comment:
It seems like we don't have consistency in whether exported functions start with upper or lower case, e.g. `RegisterTarget...` vs `registerTarget...`. Do we want to pick one to use for all new code?

Reply:
It should be RegisterXxx. Upper case like static functions

Reply:
@RaananW do you know if we can enforce this with ESLint rules?